pkg/storaged/storage-controls.jsx

```diff
@@ -284,10 +291,25 @@ class StorageMultiAction extends React.Component {
     }
 }
 
+class StorageUsageBar  extends React.Component {
+    render() {
+        var fraction = this.props.used / this.props.total;
```

Comment:
As discussed above, would be more robust to just return `if (!this.props.total)`, or render something that shows "N/A" or so. Then we don't need a "magic" 1 value for unknown sizes.

Reply:
Done.